closure/repositories.bzl

```diff
@@ -681,79 +682,38 @@ def com_google_jsinterop_annotations():
       ],
   )
 
+def com_google_protobuf():
+  native.http_archive(
+    name = "com_google_protobuf",
+    sha256 = "0cc6607e2daa675101e9b7398a436f09167dffb8ca0489b0307ff7260498c13c",
+    strip_prefix = "protobuf-3.5.0",
+    urls = [
+      "https://mirror.bazel.build/github.com/google/protobuf/archive/v3.5.0.tar.gz",
```

Comment:
This bumps the protoc version from 3.3 to 3.5. (Because of some changes in the JSPB runtime, this requires a similar bump to com_google_protobuf_js below.) If you prefer, I can stick with 3.3 in this PR and bump the versions later.